src/System.Private.Xml/tests/XmlDocument/XmlAttributeCollectionTests/InsertAfterTests.cs

```diff
@@ -230,6 +231,7 @@ public void InsertAfterRemovesDupRefAttrAtTheEnd()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)] //Fix for this edge case was only made in NetCore
```

Comment:
Yeah I found that one -- it was clearly a conscious change, I guess we update the tests to match.